doc/source/whatsnew/v0.25.0.rst

```diff
@@ -158,29 +158,64 @@ cause a ``SparseSeries`` or ``SparseDataFrame`` to be returned, as before.
 Increased minimum versions for dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions.
-Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`24942`).
-If installed, we now require:
-
-+-----------------+-----------------+----------+
-| Package         | Minimum Version | Required |
-+=================+=================+==========+
-| beautifulsoup4  | 4.4.1           |          |
-+-----------------+-----------------+----------+
-| openpyxl        | 2.4.0           |          |
-+-----------------+-----------------+----------+
-| pymysql         | 0.7.9           |          |
-+-----------------+-----------------+----------+
-| pytz            | 2015.4          |          |
-+-----------------+-----------------+----------+
-| sqlalchemy      | 1.1.4           |          |
-+-----------------+-----------------+----------+
-| xlsxwriter      | 0.7.7           |          |
-+-----------------+-----------------+----------+
-| xlwt            | 1.0.0           |          |
-+-----------------+-----------------+----------+
-| pytest (dev)    | 4.0.2           |          |
-+-----------------+-----------------+----------+
+Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions (issue:`25725`, :issue:`24942`, :issue:`25752`).
+Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`25554`).
+
+Below is an overview of the minimal versions, as required resp. recommended. Optional libraries below the lowest tested version may still work, but are not considered supported.
+
++-----------------+-----------------+-----------------------+
+| Package         | Minimum Version | Comment               |
++=================+=================+=======================+
+| numpy           | 1.13.3          | Required              |
++-----------------+-----------------+-----------------------+
+| pytz            | 2015.4          | Required              |
++-----------------+-----------------+-----------------------+
+| bottleneck      | 1.2.1           | Required if installed |
++-----------------+-----------------+-----------------------+
+| numexpr         | 2.6.2           | Required if installed |
++-----------------+-----------------+-----------------------+
+| beautifulsoup4  | 4.4.1           | Lowest tested version |
```

Comment:
remove beautifulsoup, blosc, gcfs, junja2, lxml, psycopg2, pymsql, s3fs, cython and pytest

Reply:
pytest was explicitly added to this table by #23519.

I added all the libraries from https://pandas-docs.github.io/pandas-docs-travis/install.html#dependencies. I think this is a worthwhile overview to have *somewhere* (maybe not the whatsnew). Thoughts, @datapythonista?